tidb-cloud/monitor-tidb-cluster.md

```diff
@@ -26,22 +26,29 @@ You can see the current status of each running cluster on the cluster page.
 
 ### TiDB node status
 
+> **Note:**
+>
+> The TiDB node status is only available for Dedicated Tier clusters.
+
 | TiDB node status | Description |
 |:--|:--|
-| **Normal** | Normal running |
-| **Creating** | Creating the node |
+| **Available** | The TiDB node is healthy and available |
```

Comment:
To be consistent with Cluster Status
```suggestion
| **AVAILABLE** | The TiDB node is healthy and available. |
```

Reply:
> To be consistent with Cluster Status

<img width="698" alt="image" src="https://user-images.githubusercontent.com/60599231/203217953-63562bee-9f79-4811-b5ac-77364d086ad1.png">